samples/sample_encode/src/sample_encode.cpp

```diff
@@ -277,7 +278,34 @@ mfxStatus ParseInputString(msdk_char* strInput[], mfxU8 nArgNum, sInputParams* p
             pParams->FileInputFourCC = MFX_FOURCC_P010;
             pParams->EncodeFourCC = MFX_FOURCC_P010;
         }
-        else if (0 == msdk_strcmp(strInput[i], MSDK_STRING("-ec::p010")))
+		else if (0 == msdk_strcmp(strInput[i], MSDK_STRING("-ayuv")))
```

Comment:
alignment is wrong

Reply:
it is correct - Github displays tab length incorrectly